lib/rubocop/cop/style/hash_syntax.rb

```diff
@@ -87,7 +87,7 @@ def hash_rockets_check(pairs)
         end
 
         def ruby19_no_mixed_keys_check(pairs)
-          if force_hash_rockets?
+          if force_hash_rockets? pairs
```

Comment:
I would add parentheses around `pairs` here.

Reply:
Yes, I should have paid more attention to the surrounding code. 
Fixed that (plus merged current master)